src/app/components/listing/details/ics-family/ics-family.jsx

```diff
@@ -140,137 +170,152 @@ function ChplIcsFamily(props) {
-                        <TableCell>{ l.certificationStatus.name }</TableCell>
-                        <TableCell>
-                          <List>
-                            { l.parents.map((p) => (
-                              <ListItem key={p.chplProductNumber}>
-                                {p.chplProductNumber}
-                              </ListItem>
-                            ))}
-                          </List>
-                        </TableCell>
-                        <TableCell>
-                          <List>
-                            { l.children.map((c) => (
-                              <ListItem key={c.chplProductNumber}>
-                                {c.chplProductNumber}
-                              </ListItem>
-                            ))}
-                          </List>
-                        </TableCell>
-                      </TableRow>
-                    ))}
-                  </TableBody>
-                </Table>
-              </figcaption>
-            </figure>
-            { !isLoading
-              && (
-                <ChplLink
-                  href={compare}
-                  text="Compare"
-                  external={false}
-                />
-              )}
-            { isShowingListingDetails
-              && (
-                <>
-                  <Typography variant="h5">{ listing?.chplProductNumber }</Typography>
-                  <Typography>
-                    <strong>Developer:</strong>
-                    <ChplLink
-                      href={`#/organizations/developers/${listing?.developer.id}`}
-                      text={listing?.developer.name}
-                      external={false}
-                    />
-                  </Typography>
-                  <Typography>
-                    <strong>Product:</strong>
-                    {' '}
-                    { listing?.product.name }
-                  </Typography>
-                  <Typography>
-                    <strong>Version:</strong>
-                    {' '}
-                    { listing?.version.name }
-                  </Typography>
-                  <Typography>
-                    <strong>Certification Status:</strong>
-                    {' '}
-                    { listing?.certificationStatus.name }
-                  </Typography>
-                  { listing?.id && (id !== listing.id)
-                    && (
+          <Card>
+            <CardContent>
+              <figure>
+                <div className={classes.cardContainer}>
+                  <div className={classes.directionContainer}>
+                    <Typography> Select a listing to the right to view more information. You can also click and drag to scroll through the listings.</Typography>
+                    <Typography>
+                      To compare all listings together, please select
+                      {' '}
+                      {' '}
+                      <Link
```

Comment:
This should be a ChplLink

Reply:
If I use the ChplLink then we can not alter the flex that causing the link to jump down a line

Reply:
The problem is that without the "ChplLink" component we don't have a way of tracking analytic events on clicks. We can track navigation, but we won't be able to have an event that says, for instance, "compared from ICS Relationship figure"